core/src/main/java/com/netease/arctic/CommonUnifiedCatalog.java

```diff
@@ -41,16 +41,29 @@ public class CommonUnifiedCatalog implements UnifiedCatalog {
   private Map<TableFormat, FormatCatalog> formatCatalogs = Maps.newHashMap();
   private final Map<String, String> properties = Maps.newHashMap();
 
+  private TableMetaStore tableMetaStore;
+
   public CommonUnifiedCatalog(
       Supplier<CatalogMeta> catalogMetaSupplier, Map<String, String> properties) {
     CatalogMeta catalogMeta = catalogMetaSupplier.get();
     CatalogUtil.mergeCatalogProperties(catalogMeta, properties);
     this.meta = catalogMeta;
+    this.tableMetaStore = CatalogUtil.buildMetaStore(catalogMeta);
     this.properties.putAll(properties);
     this.metaSupplier = catalogMetaSupplier;
     initializeFormatCatalogs();
   }
 
+  @Override
+  public String metastoreType() {
+    return meta.getCatalogType();
+  }
+
+  @Override
+  public TableMetaStore authenticationContext() {
```

Comment:
When I try to complete this #2344 Flink Unified Catalog(Iceberg format table), I find out that this API is aslo needed.